### DISCUSSION TITLE
Proxmox HAOS hardware: Pi4,  i5-6500T, Celeron 5105, 5095 or J4005

### DISCUSSION THREAD

**Discussion by Parahassio** [tteck/Proxmox#1257] [General]
Hi, I am currently using a RPi4 with 4GB RAM and an SSD.
Some sensors, nothing too fancy, glances, grafana, mqtt, zigbee2mqtt, nodered and wireguard.
I would like to also use Vaultwarden, Adguard home, Qbittorrent once at moth and possibly jellyfin without transcoding if I upgrade.

I do not want to experiment too much with my RPi, beacuse it is the core of the house.

I am specially worried about the power comsupton, as at this moment it is almost nothing.

I am looking at at options to upgrade. I am clear in 16 GB RAM, but no so much in the processor.
  - HP Elitedesk 800 G3 DM Intel i5-6500T RAM 16GB Disco 250GB for 158€ in Amazon Renewed (1 year of warranty). 
  -  Beelink Mini PC Mini-S con 11th Gen Celeron N5095 220€.
  - NUC7CJYHN UCFF J4005
 
 What do you think it will be the Power Consumtion with the above services? Is it a great update?

Thanks for your help!

**Comment by tteck** [comment]
At that price, it would be difficult to resist the Elitedesk, so that's my preferred option.

**Reply by bam27beer** [reply]
im using the elite desk ,though with 32gb ,with blueiris on windows 10 with 4x cams ,
and about finish setting up true nas ,then the arrrs ,no issues with home assistant restored from a pi via a back up , and bought a terramaster 5bay harddisk enclosure ,usb c - for jbod for 90 notes ,2nd hand

**Reply by Parahassio** [reply]
Thanks for your comments! I ended up buying an Elitedesk 800 G4 with 8600T, 16 Gb RAM and 256 Gb NMVe for 200€, with a lot of hours but in good state. 
I did not have time to test it (personal problems at the moment) but I am pretty happy about the deal. 

Thanks for your help!
